### PR TITLE
More Chaplain Religions And Clothing

### PR DESCRIPTION
**Pull request by Drowzyleaf**
As it turns out, religion code is so simple even someone with three brain cells such as me can figure things out.
Xenophilism was added because I noticed we had a religion for literally every antag except xenos. Includes one xeno costume.
Dudeism was added because I was reminded of it's existance by watching the Big Lebowski; it's actually a fully-fleshed out religion. Includes hawaiian shirt.
Kemetism is another name for Egyptian Mythology's religion. Includes pharaoh hat.
Primitivism is grug's favorite religion, you'll surely impress tribe leader. Includes shorts and leather mantle.
Degeneracy gives you cursed cat ears, a schoolgirl outfit, and eternal shame. I'm sure that's all you need to hear about the degenerate religion. 

Also includes added clothing to a few of the other religions where I felt it was fitting, like giving the druid coat to the druid, business suits to the capitalists, and basketball shorts to the Slam Jam.

:cl:
 * rscadd: Added five more religions: Xenophilism, Dudeism, Kemetism, Primitivism, and Degeneracy.
 * rscadd: Added more clothing to other religions, including ancom, ianism, nazi, american, shinto, wizard, security, capitalism, anarcho-capitalism, death, druid, basketball, and revolution.

### CODE REVIEW

File: code/datums/religions.dm [hunk 1]
```diff
@@ -1420,3 +1446,69 @@
...
+	female_adept = "Hemet-Netjer"
+	keys = list("kemetism", "egypt", "egyptian")
+
+/datum/religion/egypt/equip_chaplain(var/mob/living/carbon/human/H)
+	H.equip_or_collect(new /obj/item/clothing/head/pharaoh(H), slot_head)
+
+/datum/religion/anprim
+	name = "Primitivism"
+	deity_name = "Grug"
+	bible_name = "Blank Book" //illiteracy is radical
```
Comment (ShiftyRail) [inline code/datums/religions.dm]: Why not "Industrial Society and Its Future" ?

Comment (Drowzyleaf) [reply code/datums/religions.dm]: yeah that's reasonable, i was havin trouble comin up with the anprim book but goin with the obvious is probably good

File: code/datums/religions.dm [hunk 2]
```diff
@@ -1420,3 +1446,69 @@
...
+	female_adept = "Hemet-Netjer"
+	keys = list("kemetism", "egypt", "egyptian")
+
+/datum/religion/egypt/equip_chaplain(var/mob/living/carbon/human/H)
+	H.equip_or_collect(new /obj/item/clothing/head/pharaoh(H), slot_head)
+
+/datum/religion/anprim
+	name = "Primitivism"
+	deity_name = "Grug"
+	bible_name = "Industrial Society and It's Future"
```
Comment (kilozombie) [inline code/datums/religions.dm]: >it's

Comment (Drowzyleaf) [reply code/datums/religions.dm]: good grammar is for suckers

Comment (Drowzyleaf) [reply code/datums/religions.dm]: fixed

**Comment by anunknownentity03** [comment]
previews?

**Comment by thestubborn** [comment]
>Xenophilism was added because I noticed we had a religion for literally every antag except xenos. Includes one xeno costume.

finally, i've been trying to get that gimmic off the ground for ages

**Comment by DamianX** [comment]
[Remember to update the wiki](http://ss13.moe/wiki/index.php/Chaplain)

**Pull request merged**